contracts/LinkedList.sol

```diff
@@ -72,6 +75,20 @@ contract LinkedList {
         return size;
     }
 
+    function contains(string memory data) public view returns(bool) {
+        uint256 hash = getDataHash(data);
+
+        return compareStrings(list[hash].data, "");
```

Comment:
why are you comparing to `""`?

Reply:
to check if i found an empty string location. i made a mistake and added a ! in front of the return so that it read if `list[hash].data != ""` mark as not found.